api/api/routers/mcp/_mcp_models.py

```diff
@@ -40,38 +40,61 @@ class Link(BaseModel):
 
 
 class ConciseLatestModelResponse(BaseModel):
-    id: str
-    currently_points_to: str
+    id: str = Field(description="The id of the model")
+    currently_points_to: str = Field(
+        description="The id of the model that the latest version of the agent is currently pointing to",
+    )
 
 
 class ConciseModelResponse(BaseModel):
-    id: str
-    display_name: str
-    supports: list[str]
-    quality_index: int
-    cost_per_input_token_usd: float
-    cost_per_output_token_usd: float
-    release_date: str
+    class ModelSupports(BaseModel):
```

Comment:
Ah ! I also implemented a similar case in https://github.com/WorkflowAI/WorkflowAI/pull/601/files#diff-e7e24a283807cea36415003b404fd407dc72c8a00edc3b102f0337d6b93af238 when I added reasoning